tests/io/test_DataAccess_FouFile.py

```diff
@@ -0,0 +1,135 @@
+import sys
+from contextlib import contextmanager
+from io import StringIO
+
+import numpy
+import pytest
+
+from dfastmi.io.FouFile import FouFile
+
+
+def open_fou_file() -> FouFile:
+    filename = "tests/files/e02_f001_c011_simplechannel_fou.nc"
+    return FouFile(filename)
+
+
+@pytest.fixture
+def fou_file() -> FouFile:
+    return open_fou_file()
+
+
+@contextmanager
+def captured_output():
+    new_out, new_err = StringIO(), StringIO()
+    old_out, old_err = sys.stdout, sys.stderr
+    try:
+        sys.stdout, sys.stderr = new_out, new_err
+        yield sys.stdout, sys.stderr
+    finally:
+        sys.stdout, sys.stderr = old_out, old_err
+
+
+class Test_data_access_read_face_variable:
+    @pytest.mark.parametrize(
+        "data_file, dataref",
+        [
+            (open_fou_file(), 1.2839395356652856),
+        ],
+    )
+    def test_x_velocity(self, data_file: FouFile, dataref: float):
+        """
+        Testing x_velocity.
+        """
+        datac = data_file.x_velocity()
+        assert datac[1] == dataref
+
+    @pytest.mark.parametrize(
+        "data_file, dataref",
+        [
+            (open_fou_file(), 0.0001568670009850459),
+        ],
+    )
+    def test_y_velocity(self, data_file: FouFile, dataref: float):
+        """
+        Testing y_velocity.
+        """
+        datac = data_file.y_velocity()
+        assert datac[1] == dataref
+
+    @pytest.mark.parametrize(
+        "data_file, dataref",
+        [
+            (open_fou_file(), 3.89449840620317),
+        ],
+    )
+    def test_water_depth(self, data_file: FouFile, dataref: float):
+        """
+        Testing water_depth.
+        """
+        datac = data_file.water_depth()
+        assert datac[1] == dataref
+
+    def test_read_face_variable_04(self, fou_file: FouFile):
+        """
+        Testing read_face_variable: variable by standard name.
+        """
+        varname = "sea_surface_height"
+        datac = fou_file.read_face_variable(varname)
+        dataref = 3.887132830889389
+        assert datac[1] == dataref
+
+    def test_read_face_variable_05(self, fou_file: FouFile):
+        """
+        Testing read_face_variable: variable by long name.
+        """
+        varname = "Last 001: water level, last values"
+        datac = fou_file.read_face_variable(varname)
+        dataref = 3.887132830889389
+        assert datac[1] == dataref
+
+    def test_read_face_variable_06(self, fou_file: FouFile):
```

Comment:
so this should always throw an exception?

Reply:
discussed, renamed so more clear and readable